nix/ec2.nix

```diff
@@ -175,6 +175,17 @@ in
 
   options = {
 
+    deployment.ec2.profile = mkOption {
+      default = null;
```

Comment:
The default value should be either an empty string or just "default" but not _null_ value

`error: The option value `deployment.ec2.profile' in `/nix/store/ydszllb0yml02kzi8m4l5sp5mk6z0h3i-nixops-aws-1.7pre0_abcdef/share/nix/nixops-aws/ec2.nix' is not of type `string'.
`